test/utils/deleteRoles.js

```diff
@@ -0,0 +1,25 @@
+const firestore = require("../../utils/firestore");
+const userCollection = firestore.collection("users");
+
+/**
+ * @return {boolean} success - are roles deleted or not
+ */
+module.exports = async (userId = null, rolesToBeDeleted = [], deleteRolesObject = false) => {
+  if (!userId) return false;
```

Comment:
Is there any not include this line under try block

Reply:
Since I know that this line won't ever throw an error therefore felt no need to keep it inside the try
But there is also no harm in keeping it inside the try